staging/src/k8s.io/apiserver/pkg/cel/library/cidr_test.go

```diff
@@ -151,11 +151,21 @@ func TestCIDR(t *testing.T) {
 			expr:         `cidr("192.168.0.0/24").containsCIDR(cidr("192.168.0.0/25"))`,
 			expectResult: trueVal,
 		},
+		{
```

Comment:
Does it make sense to add coverage for IPv6 subnets?
```
should succeed:
cidr("2001:dbb::/64").containsCIDR(cidr("2001:dbb::1/128"))
cidr("2001:dbb::/64").containsCIDR(cidr("2001:dbb::/120"))
should fail:
cidr("2001:dbb::/64").containsCIDR(cidr("2002:aaa::1/128"))
cidr("2001:dbb::/64").containsCIDR(cidr("2002:aaa::/100"))
```
 @aojea  WDYT?

Reply:
yes, we should try to have coverage on both families because there are always subtle problems that this will catch, but it can be a follow up or a task for new contributors, no need to block on that